.github/workflows/ui-tests/cypress/integration/create_spec.js

```diff
@@ -28,6 +28,53 @@ describe('User can navigate to create account screen', function() {
     })
 })
 
+describe('Signup is unsuccessful with long username', function() {
+    const invalidFeedback = Cypress.env('messages').invalidFeedback
+
+    beforeEach(function() {
+        const user = {
+            username: '0123456789abcdefghijklmnopqrstuvwxyz',
```

Comment:
making the username random would keep it consistent with other tests, you could do a random string of length or a random number of at least 15 digits. 
example:
```javascript
username: randomString(15, 100)
or
username: invalid_user_${randomNumber(2,100)}
```
what do you think?

Reply:
That's a good point, but I think a test with lengths could be dangerous (what if it only fails with 16 characters, so 1 out of 86 tests fail and we assume it's just flakiness?)

I could do a random string of 16 digits, since that's the first failure case. Or is it possible to loop with multiple values for the length? is randomString defined somewhere, or do I need to implement it myself?

Reply:
that's fair!  in that case, you can keep as is or just attach random number that would result in a 16 char username (keeping as is seems simplest 😄 )

random num example:
https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/f1039dc57df89aa74e98ec770e01819ac117a08f/.github/workflows/ui-tests/cypress/integration/create_spec.js#L31

```javascript
describe('Signup is unsuccessful with long username', function() {
    const invalidFeedback = Cypress.env('messages').invalidFeedback
    const uuid = () => Cypress._.random(1e7, 1e8)
    const longUsername = `invalid_${uuid}`
    beforeEach(function() {
        const user = {
            username: longUsername
            firstName: 'Tom',
            lastName: 'Nook',
            password: 'bells'
        }
        cy.createAccount(user)
    })

```